src/telemetry.ts

```diff
@@ -84,6 +84,7 @@ export default class Reporter implements vscode.Disposable {
   getMxEvent(event: TelemetryEvent): Mixpanel.Event {
```

Comment:
figure out why the `auth_started` event does not have activity bar as the event source (in recorded events)